lib/govuk_tech_docs/api_reference/templates/schema.html.erb

```diff
@@ -6,18 +6,18 @@
 <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th><th>Schema</th></tr>
 </thead>
 <tbody>
-<% properties.each do |property| %>
+<% properties.each do |property_name, property_attributes| %>
```

Comment:
Minor: the commit message confusingly refers to a `attribute_name`, I assume it should refer to `property_name` instead.

Reply:
Sure - I've changed it (felt a bit weird editing someone else's commit)